[Bug]: No signature of method is applicable for argument types

### Platform

Android 11 (Emulator)

### Plugin

share_plus

### Version

6.3.3

### Flutter SDK

3.10.0-1.3.pre

### Steps to reproduce

1. flutter run

### Code Sample

_No response_

### Logs

```shell
FAILURE: Build failed with an exception.

* Where:
Build file 'C:\Users\Desktop\AppData\Local\Pub\Cache\hosted\pub.dev\share_plus-6.3.3\android\build.gradle' line: 27

* What went wrong:
A problem occurred evaluating project ':share_plus'.
> No signature of method: build_982riom3gsel1z49qvhv3msqn.android() is applicable for argument types: (build_982riom3gsel1z49qvhv3msqn$_run_closure2) values: [build_982riom3gsel1z49qvhv3msqn$_run_closure2@709d9b67]

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 28s
Exception: Gradle task assembleDebug failed with exit code 1
```


### Flutter Doctor

```shell
[√] Flutter (Channel beta, 3.10.0-1.3.pre, on Microsoft Windows [Version 10.0.22624.1616], locale en-US)
    • Flutter version 3.10.0-1.3.pre on channel beta at C:\Users\Desktop\fvm\versions\beta
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision d11aff97d2 (7 days ago), 2023-04-19 11:11:05 -0700
    • Engine revision a645b36a9b
    • Dart version 3.0.0 (build 3.0.0-417.2.beta)
    • DevTools version 2.23.1

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 33.0.0-rc4)
    • Android SDK at C:\Users\Desktop\AppData\Local\Android\sdk
    • Platform android-33, build-tools 33.0.0-rc4
    • Java binary at: C:\Program Files\Android\Android Studio\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Build Tools 2019 16.11.26)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools
    • Visual Studio Build Tools 2019 version 16.11.33529.622
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2022.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)

[√] VS Code (version 1.77.3)
    • VS Code at C:\Users\Desktop\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.62.0

[√] Connected device (4 available)
    • sdk gphone x86 (mobile) • emulator-5554 • android-x86    • Android 11 (API 30) (emulator)
    • Windows (desktop)       • windows       • windows-x64    • Microsoft Windows [Version 10.0.22624.1616]
    • Chrome (web)            • chrome        • web-javascript • Google Chrome 112.0.5615.138
    • Edge (web)              • edge          • web-javascript • Microsoft Edge 112.0.1722.58

[√] Network resources
    • All expected network resources are available.

• No issues found!
```


### Checklist before submitting a bug

- [X] I Google'd a solution and I couldn't find it
- [X] I searched on StackOverflow for a solution and I couldn't find it
- [X] I read the README.md file of the plugin
- [X] I'm using the latest version of the plugin
- [X] All dependencies are up to date with `flutter pub upgrade`
- [X] I did a `flutter clean`
- [X] I tried running the example project

--- a/packages/android_alarm_manager_plus/android/build.gradle
+++ b/packages/android_alarm_manager_plus/android/build.gradle
@@ -23,15 +23,12 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 31
-    namespace 'dev.fluttercommunity.plus.androidalarmmanager'
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
+
     defaultConfig {
         minSdkVersion 16
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+
     lintOptions {
         disable 'InvalidPackage'
     }

--- a/packages/android_alarm_manager_plus/example/android/app/build.gradle
+++ b/packages/android_alarm_manager_plus/example/android/app/build.gradle
@@ -27,7 +27,6 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     compileSdkVersion 33
-    namespace 'com.example.example'
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'

--- a/packages/android_intent_plus/android/build.gradle
+++ b/packages/android_intent_plus/android/build.gradle
@@ -23,15 +23,16 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 31
-    namespace 'dev.fluttercommunity.plus.androidintent'
 
     defaultConfig {
         minSdkVersion 16
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+
     lintOptions {
         disable 'InvalidPackage'
     }
+
     testOptions {
         unitTests.includeAndroidResources = true
     }

--- a/packages/android_intent_plus/example/android/app/build.gradle
+++ b/packages/android_intent_plus/example/android/app/build.gradle
@@ -26,7 +26,6 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     compileSdkVersion 31
-    namespace 'io.flutter.plugins.androidintentexample'
 
     lintOptions {
         disable 'InvalidPackage'

--- a/packages/battery_plus/battery_plus/android/build.gradle
+++ b/packages/battery_plus/battery_plus/android/build.gradle
@@ -24,12 +24,12 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 33
-    namespace 'dev.fluttercommunity.plus.battery'
 
     defaultConfig {
         minSdkVersion 16
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+
     lintOptions {
         disable 'InvalidPackage'
     }

--- a/packages/battery_plus/battery_plus/example/android/app/build.gradle
+++ b/packages/battery_plus/battery_plus/example/android/app/build.gradle
@@ -27,7 +27,6 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     compileSdkVersion 33
-    namespace 'io.flutter.plugins.batteryexample.example'
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'

--- a/packages/connectivity_plus/connectivity_plus/android/build.gradle
+++ b/packages/connectivity_plus/connectivity_plus/android/build.gradle
@@ -23,12 +23,6 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 31
-    namespace 'dev.fluttercommunity.plus.connectivity'
-
-    compileOptions {
-      sourceCompatibility JavaVersion.VERSION_1_8
-      targetCompatibility JavaVersion.VERSION_1_8
-    }
 
     defaultConfig {
         minSdkVersion 16

--- a/packages/connectivity_plus/connectivity_plus/example/android/app/build.gradle
+++ b/packages/connectivity_plus/connectivity_plus/example/android/app/build.gradle
@@ -26,7 +26,6 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     compileSdkVersion 31
-    namespace 'io.flutter.plugins.connectivityexample'
 
     lintOptions {
         disable 'InvalidPackage'

--- a/packages/device_info_plus/device_info_plus/android/build.gradle
+++ b/packages/device_info_plus/device_info_plus/android/build.gradle
@@ -26,12 +26,12 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 33
-    namespace 'dev.fluttercommunity.plus.device_info'
 
     defaultConfig {
         minSdkVersion 16
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+
     lintOptions {
         disable 'InvalidPackage'
     }

--- a/packages/device_info_plus/device_info_plus/example/android/app/build.gradle
+++ b/packages/device_info_plus/device_info_plus/example/android/app/build.gradle
@@ -27,7 +27,6 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     compileSdkVersion 33
-    namespace 'io.flutter.plugins.deviceinfoexample.example'
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'

--- a/packages/network_info_plus/network_info_plus/android/build.gradle
+++ b/packages/network_info_plus/network_info_plus/android/build.gradle
@@ -26,12 +26,12 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 33
-    namespace 'dev.fluttercommunity.plus.network_info'
 
     defaultConfig {
         minSdkVersion 16
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+
     lintOptions {
         disable 'InvalidPackage'
     }

--- a/packages/network_info_plus/network_info_plus/example/android/app/build.gradle
+++ b/packages/network_info_plus/network_info_plus/example/android/app/build.gradle
@@ -26,7 +26,6 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     compileSdkVersion 33
-    namespace 'dev.fluttercommunity.plus.network_info_plus_example'
 
     lintOptions {
         disable 'InvalidPackage'

--- a/packages/package_info_plus/package_info_plus/android/build.gradle
+++ b/packages/package_info_plus/package_info_plus/android/build.gradle
@@ -27,12 +27,12 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 31
-    namespace 'dev.fluttercommunity.plus.packageinfo'
 
     defaultConfig {
         minSdkVersion 16
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+
     lintOptions {
         disable 'InvalidPackage'
     }

--- a/packages/package_info_plus/package_info_plus/example/android/app/build.gradle
+++ b/packages/package_info_plus/package_info_plus/example/android/app/build.gradle
@@ -26,7 +26,6 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     compileSdkVersion 31
-    namespace 'io.flutter.plugins.packageinfoexample'
 
     lintOptions {
         disable 'InvalidPackage'

--- a/packages/sensors_plus/sensors_plus/android/build.gradle
+++ b/packages/sensors_plus/sensors_plus/android/build.gradle
@@ -26,12 +26,12 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 31
-    namespace 'dev.fluttercommunity.plus.sensors'
 
     defaultConfig {
         minSdkVersion 16
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+
     lintOptions {
         disable 'InvalidPackage'
     }

--- a/packages/sensors_plus/sensors_plus/example/android/app/build.gradle
+++ b/packages/sensors_plus/sensors_plus/example/android/app/build.gradle
@@ -26,7 +26,6 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     compileSdkVersion 31
-    namespace 'io.flutter.plugins.sensorsexample'
 
     lintOptions {
         disable 'InvalidPackage'

--- a/packages/share_plus/share_plus/android/build.gradle
+++ b/packages/share_plus/share_plus/android/build.gradle
@@ -26,12 +26,12 @@ apply plugin: 'kotlin-android'
 
 android {
   compileSdkVersion 33
-  namespace 'dev.fluttercommunity.plus.share'
 
   defaultConfig {
     minSdkVersion 16
     testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
   }
+
   lintOptions {
     disable 'InvalidPackage'
   }

--- a/packages/share_plus/share_plus/example/android/app/build.gradle
+++ b/packages/share_plus/share_plus/example/android/app/build.gradle
@@ -26,7 +26,6 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     compileSdkVersion 33
-    namespace 'io.flutter.plugins.shareexample'
 
     lintOptions {
         disable 'InvalidPackage'

